pkg/api/vlabs/validate.go

```diff
@@ -302,14 +302,20 @@ func (a *Properties) validateOrchestratorProfile(isUpdate bool) error {
 	return nil
 }
 
+// Validate implements APIObject
```

Comment:
I don't understand this comment?

Reply:
Looks like some sort of copy pasta from here: https://github.com/Azure/acs-engine/blob/master/pkg/api/vlabs/validate.go#L97

Will remove.